ui/webui/test/machine_install.py

```diff
@@ -54,6 +55,12 @@ def _get_free_port(self, start_port):
                 return port
             port = port + 1
 
+    def _create_disk_image(self, size=15):
+        name = f"disk-anaconda-{self.label}"
+        _, image = tempfile.mkstemp(suffix='.qcow2', prefix=name, dir=self.run_dir)
```

Comment:
I guess you use mkstemp to get some completely random name for the disk?
Generally self.label should be unique, and it is in cockpit, it's a bug in anaconda-webui test code that this is not unique. @jelly was taking a look on this.


Maybe add a comment to revert to using self.label as prefix for the disk instead of name when this will be fixed?

Reply:
I'd prefer to be safe here using random name. Also I can imagine creating more than 1 disk this way for a test / vm (I am already using it locally when developing). Maybe we should even use this in `add_disk()` to be able to add more then a single disk in scope of a single vm? I think self.label is per-vm thing?

Reply:
Maybe we can consolidate creating disk(s) for testing vm and adding disks with add_disk: https://github.com/rhinstaller/anaconda/pull/4731 ?

Reply:
@KKoukiou would you be OK with this version, or https://github.com/rhinstaller/anaconda/pull/4731 ?